gms_app/migrations/0008_alter_plantpreset_options.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 5.0.6 on 2024-06-05 02:46
```

Comment:
Normally, I try for one migration per app in a PR, but since we're early in development and will likely squash when the initial models have settled out, I figure multiple migrations are ok since they're for different models and that might help make figuring things out clearer